crates/pink/src/runtime.rs

```diff
@@ -144,6 +145,7 @@ impl Config for PinkRuntime {
     type AddressGenerator = Pink;
     type MaxCodeLen = MaxCodeLen;
     type MaxStorageKeyLen = MaxStorageKeyLen;
+    type UnsafeUnstableInterface = ConstBool<false>;
```

Comment:
@kvinwang Caution

Reply:
LGTM. We never want contracts to call unstable interfaces.